manubot/pandoc_filter/cite.py

```diff
@@ -0,0 +1,166 @@
+"""
+Preliminary testing command:
+
+pandoc \
+  --to=plain \
+  --standalone \
+  --filter=pandoc-manubot-cite \
+  --filter pandoc-citeproc \
+  manubot/pandoc_filter/tests/input-with-cites.md
+
```

Comment:
Is there a difference between `--filter=` and `--filter ` (with a space)?

Reply:
No